### PR TITLE
Ebru.OddEven

### PR DESCRIPTION
**Pull request by EbruErol**
OddEven task

### CODE REVIEW

**Comment by EbruErol** [comment]
Please check the code.

File: src/ebru/OddEven.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+            System.out.println(isOdd(0));
+
+            System.out.println(isEven(0));
+        }
+
+        public static boolean isOdd(int num) {
+
+            return (num % 2 != 0) ? true : false;
+        }
```
Comment (merverdogan) [inline src/ebru/OddEven.java:15]: comment

File: src/ebru/OddEven.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+        public static boolean isEven(int num) {
+
+            return !isOdd(num);
+        }
+    }
```
Comment (MariiaBorovykova) [inline src/ebru/OddEven.java:21]: new comment

File: src/ebru/OddEven.java (hunk 1 above)
Comment (EbruErol) [reply src/ebru/OddEven.java:15]: comment reviewed.

File: src/ebru/OddEven.java (hunk 2 above)
Comment (EbruErol) [reply src/ebru/OddEven.java:21]: comment reviewed.

**Pull request merged**